partcad-cli/src/partcad_cli/click/command.py

```
@@ -0,0 +1,156 @@
+cli.context_settings = {
+    "show_default": True,
+    # terminal_width
+    # max_content_width
+    # "": StderrContext,
+}
+
+
+@cli.result_callback()
+def process_result(result, v, q, no_ansi, p, format):
+    if pc.logging.had_errors:
+        raise click.Abort()
```

Ruff (0.8.0)

Unnecessary parentheses on raised exception

Remove unnecessary parentheses

(RSE102)